### PR TITLE
[api] change format of latest_update to int64

### PR DESCRIPTION
**Pull request by Andrew7234**
Follow up of https://github.com/oasisprotocol/nexus/pull/496#discussion_r1309539679

### CODE REVIEW

**Review by lukaw3d — APPROVED** [review]

File: api/spec/v1.yaml [hunk 1]
```diff
@@ -1175,9 +1175,10 @@ components:
           description: The RFC 3339 formatted consensus time of when the most recently indexed block was produced.
           example: *iso_timestamp_2
         latest_update:
```
Comment (mitjat) [inline api/spec/v1.yaml:1177]: Optional: The field implies an absolute timestamp to me. You might want to consider `latest_update_age[_ms]` or similar.

**Review by mitjat — APPROVED** [review]
Thank you!

File: api/spec/v1.yaml (hunk 1 above)
Comment (lukaw3d) [reply api/spec/v1.yaml:1177]: I agree. Naming-wise I vote "time_since_latest_block"

Comment (lukaw3d) [reply api/spec/v1.yaml:1177]: oh, too easily confused with time since chain produced a block.

Comment (Andrew7234) [reply api/spec/v1.yaml]: updated to `latest_update_age_ms`

**Pull request merged**